docs/rst/tutorials/core/qos/mutable/mutable.rst

```diff
@@ -0,0 +1,296 @@
+The recommendation is to first complete the following tutorials:
+
+    * :ref:`Modifying Ownership and Ownership Strength QoS Policy <tutorials_qos_ownership_ownership>`
+
+    * :ref:`Writing a simple publisher and subscriber (C++) <CppPubSub>`
+
+    * :ref:`Understanding parameters <ROS2Params>`
+
+    * :ref:`Using parameters in a class (C++) <CppParamNode>`
+
+    * :ref:`Monitoring for parameter changes (C++) <MonitorParams>`
+
+Create a package
+----------------
+
+First, open a new terminal and source your Vulcanexus installation.
+
+.. code-block:: bash
+
+    source /opt/vulcanexus/humble/setup.bash
+
+
+Next lines will create a package in a new workspace folder named `vulcanexus_ws` in `~/`.
+
+.. code-block:: bash
+
+    mkdir ~/vulcanexus_ws && mkdir ~/vulcanexus_ws/src && cd ~/vulcanexus_ws/src
+    ros2 pkg create --build-type ament_cmake \
+        vulcanexus_change_mutable_qos --dependencies rclcpp
+
+
+Make sure to add the description, maintainer email and name, and license information to the `package.xml` file of the created package, as explained in the aforementioned tutorials.
+
+Write sources
+-------------
+
+First, the source file of the first publisher will be created, the Publisher 1.
+Inside the `vulcanexus_ws/src/vulcanexus_change_mutable_qos/src` directory, create a new file called `change_mutable_qos_pub1.cpp` and paste the following code within:
+
+.. literalinclude:: /resources/tutorials/core/qos/mutable/vulcanexus_change_mutable_qos/src/change_mutable_qos_pub1.cpp
+    :language: c++
+
+
+The code for the Publisher 2 is the same, just changing the Publisher 1 names for Publisher 2 names.
+Inside the `vulcanexus_ws/src/vulcanexus_change_mutable_qos/src` directory, create a new file called `change_mutable_qos_pub2.cpp` and paste the following code within:
+
+.. literalinclude:: /resources/tutorials/core/qos/mutable/vulcanexus_change_mutable_qos/src/change_mutable_qos_pub2.cpp
+    :language: c++
+
+
+The case of the subscriber is easier, as it is only need a minimal subscriber for this tutorial.
+Inside the `vulcanexus_ws/src/vulcanexus_change_mutable_qos/src` directory, create a new file called `change_mutable_qos_sub.cpp` and paste the following code within:
+
+.. literalinclude:: /resources/tutorials/core/qos/mutable/vulcanexus_change_mutable_qos/src/change_mutable_qos_sub.cpp
+    :language: c++
+
+
+Explaining the source code
+--------------------------
+
+In the case of the Publishers, the code is analogous, so here the code is going to be explained just for Publisher 1.
+For the case of the Subscriber, this tutorial is not going to explain it, as it is just the minimal subscriber, listening on the topic `/chatter`, already explained in the :ref:`Writing a simple publisher and subscriber (C++) <CppPubSub>` tutorial.
+
+For the Publisher, here not all the code is going to be explained, as the referred tutorials of the prerequisites section explain big part of it.
```

Comment:
I think it's a very good idea to point to the tutorials that explain each thing. That being said, I wouldn't even put the snippet here unless it's relevant for which we want to explain

Reply:
I erased the snippets. Done